File: Digital Business Enablement/bluegather/src/main/java/br/com/fiap/bluegather/dto/AvaliacaoDTO.java
package br.com.fiap.bluegather.dto;

import jakarta.validation.constraints.*;

import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AvaliacaoDTO {
    private Long id;

    @NotNull(message = "O campo idEvento não pode estar vazio.")
    private Long idEvento;

    @NotNull(message = "O campo idAvaliador não pode estar vazio.")
    private Long idAvaliador;

    @Positive
    @Min(1) @Max(5)
    @NotNull(message = "O campo nota não pode estar vazio.")
    private Long nota;
}

File: Enterprise Application Development/bluegather/Dtos/AvaliacaoDto.cs
using System.ComponentModel.DataAnnotations;

namespace BlueGather.Dto
{
    public class AvaliacaoDto
    {
        public long? Id { get; set; }

        [Required(ErrorMessage = "O campo idEvento não pode estar vazio.")]
        public long IdEvento { get; set; }

        [Required(ErrorMessage = "O campo idAvaliador não pode estar vazio.")]
        public long IdAvaliador { get; set; }

        [Required(ErrorMessage = "O campo nota não pode estar vazio.")]
        [Range(1, 5)]
        public long Nota { get; set; }
    }
}